goth_tests/pyproject.toml

```diff
@@ -23,7 +23,7 @@ pytest-asyncio = "^0.20.2"
 pytest-rerunfailures = "^10.3"
 #goth = "^0.11"
 # to use development goth version uncomment below
-goth =  { git = "https://github.com/golemfactory/goth.git", rev = "a6e63b875fd4b254476eaafcc20657e5a99f7f73" }
+goth =  { git = "https://github.com/golemfactory/goth.git", rev = "master" }
```

Comment:
If you want to have yagna goth tests passing you need to use this branch
https://github.com/golemfactory/goth/pull/617
(it is not merged - needs verification (you can mark it as verified and merge it, but this branch should not be deleted immediately when doing a squash merge because `yagna` master uses it))

Reply:
Merged to `master`, but now it fails because `docker-compose-hybrid` has been removed.